packages/neutrino/bin/neutrino

```diff
@@ -88,17 +88,9 @@ function run(command, presets) {
       `Ensure this module can be found relative to the root of the project.`);
   });
 
-  // Also grab any Neutrino config from package.json and merge it into the config at a higher precedence
-  if (Object.keys(config).length) {
-    api.config.merge(config);
-  }
-
-  if (args.inspect) {
-    return inspect(api.getWebpackOptions());
-  }
-
-  api[command](args)
-    .then(() => process.exit(0))
+  api.load()
+    .then(() => api.config.merge(config || {}))
```

Comment:
This should probably be `use`d prior to load for consistency:

```js
api.use(() => api.config.merge(config));
```

Also, `config` is already defaulted to an empty object on line 62.